README.md

```diff
@@ -1,5 +1,116 @@
+[![issues](https://img.shields.io/github/issues/reearth/reearth)](https://img.shields.io/github/issues/reearth/reearth)
+[![forks](https://img.shields.io/github/forks/reearth/reearth)](https://img.shields.io/github/forks/reearth/reearth)
+[![](https://img.shields.io/github/license/reearth/reearth)](https://img.shields.io/github/license/reearth/reearth)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/reearth/reearth/graphs/commit-activity)
+[![front-end contributers](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/reearth/reearth/graphs/commit-activity)
+[![GitHub release](https://img.shields.io/github/release/reearth/reearth.svg)](https://GitHub.com/reearth/reearth/releases/)
+[![GitHub tag](https://img.shields.io/github/tag/reearth/reearth.js.svg)](https://GitHub.com/reearth/reearth.js/tags/)
+[![Github all releases](https://img.shields.io/github/downloads/reearth/reearth.js/total.svg)](https://GitHub.com/reearth/reearth.js/releases/)
+[![GitHub contributors](https://img.shields.io/github/contributors/reearth/reearth.svg)](https://GitHub.com/reearth/reearth/graphs/contributors/)
+[![](https://img.shields.io/github/contributors/reearth/reearth-web?label=front-end%20contributors)](https://GitHub.com/reearth/reearth-web/graphs/contributors/)
+[![](https://img.shields.io/github/contributors/reearth/reearth-backend?label=back-end%20contributors)](https://GitHub.com/reearth/reearth-backend/graphs/contributors/)
+[![Generic badge](https://img.shields.io/badge/discussion-welcome-green.svg)](https://github.com/reearth/reearth/discussions)
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+<!-- TODO: change this link later -->
+  <a href="https://github.com/reearth/reearth">
+    <img src="./public/reearth-logo.svg" alt="Logo" width="300" height="300">
+  </a>
+
+  <p align="center">
+    Re:Earth is a state-of-the-art WebGIS with expandability through plug-ins and the ease of running in a browser.
+    <br />
+    <a  href="https://docs.reearth.io/"><strong>Documents >></strong></a>
+    &emsp;&emsp;&emsp;&emsp;
+    <a href="https://docs.reearth.io/oss"><strong>OSS Webpage >></strong></a>
+    <br />
+    <!-- TODO: here -->
+    <a href="">View Demo</a>
+    ·
+    <a href="https://github.com/reearth/reearth/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/reearth/reearth/discussions">Request Feature</a>
+  </p>
+</p>
+
+Re:Earth consist with 2 repositories. Don't forget to check our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories👍
+
+[![reearth-ui](./public/reearth-ui.png)](https://reearth.io)
+
+<!-- ABOUT THE PROJECT -->
+## Features
+🔌 Higy extensible with plugins system
+
+💻 Super handy with running on the browser
+
+💪 Support standard GIS data formats(CSV, KML, CZML, GeoJSON, shapefile)
+
+📢 Easily make the project public
+
+✨ Freely style the map
```

Comment:
If this is about being able to add markers and such perhaps this is better:
```suggestion
✨ Freely populate the map
```
If this is about being able to customize the map, like different tiles/atmosphere/etc, the original should be okay

Reply:
I meant customizing the map. Thanks!